### PR TITLE
Css added for makeing the  "Add Content page" responsive

### LINKED ISSUE #4488: The Add Content page  is not responsive  for screen size less then 766 px
The editor of Add content page has two options one for adding blocks and one for deleting.
These options are not showing if the screen size is less then  766 px,
and the sidebar is also not responsive

https://user-images.githubusercontent.com/97228857/223932019-9439a85c-7b8d-414e-91d6-4a2acf73ac92.mp4

Also if   390<= screen size<=400px   the "review_state" button is not showing

https://user-images.githubusercontent.com/97228857/223934409-3475d27b-b79f-4ca2-b770-c8b38190fa06.mp4

### PR DESCRIPTION
**Pull request by lord2anil**
Before:
https://user-images.githubusercontent.com/97228857/223932019-9439a85c-7b8d-414e-91d6-4a2acf73ac92.mp4
after:

https://user-images.githubusercontent.com/97228857/224782035-7ace259c-78fe-40b0-b00a-c5ce82de8d4e.mp4

closes #4488

### CODE REVIEW

File: news/4488.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Css added for makeing the  "Add Content page" responsive @lord2anil
```
Comment (stevepiercy) [inline news/4488.bugfix:1]:
```suggestion
CSS added to make the "Add Content page" responsive @lord2anil

```

**Review by stevepiercy — APPROVED** [review]
This should be reviewed by a core member of the Volto team.

**Comment by sneridagh** [comment]
@lord2anil adding yet another wrapper is not acceptable, and it's a breaking change. (it can break other CSS customizations, we are changing the site's structure).

We are taking care of this issue from another perspective aligned with the current roadmap and future developments.

The idea is to introduce soon a change in the containers that will use CSS container queries instead of relying in the viewport calculations. This work is part of the new theme story.

**Comment by davisagli** [comment]
Closing this PR since there is a plan to address the issue a different way. Thank you for your effort in submitting the PR.

**Pull request closed without merging**